band/test/BUILD

```diff
@@ -159,16 +159,17 @@ band_cc_android_test(
     ],
 )
 
-band_cc_android_test(
-    name = "resource_monitor_test",
-    size = "small",
-    srcs = ["resource_monitor_test.cc"],
-    data = [],
-    deps = [
-        "//band:resource_monitor",
-        "@com_google_googletest//:gtest",
-    ],
-)
+# # Deprecated
```

Comment:
Now we could uncomment this?

Reply:
I think deleting `resource_monitor_test` test is not relevant to C++17 PR. As you mentioned, how about making a seperate PR for this and issue #312 problem as solved in `band/model_analyzer.cc`?

Reply:
Sounds good!